<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #F8FAFC; /* Çok hafif grimsi beyaz, gözü yormaz */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #f5f7fa;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>Segoe UI,sans-serif</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
    /* Yazı Tipi ve Boyutu */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 24px;
    font-weight: bold;
    
    /* Renk */
    color: #333333;
    
    /* Hizalama ve Boşluk */
    qproperty-alignment: 'AlignCenter';
    margin-bottom: 10px;
    padding: 5px;
}</string>
                 </property>
                 <property name="text">
                  <string>Hoşgeldiniz</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_lessons">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #6366F1; /* Modern İndigo Mavisi */
    color: white;
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
    font-size: 14px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, sans-serif;
}
QPushButton:hover {
    background-color: #4F46E5;
    /* Hover efektinde hafif gölge (isteğe bağlı) */
}

QPushButton:pressed {
    background-color: #4338CA;
    padding-top: 14px; /* Tıklama hissi verir */
}</string>
                 </property>
                 <property name="text">
                  <string>Dersler</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_vocabulary">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #6366F1; /* Modern İndigo Mavisi */
    color: white;
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
    font-size: 14px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, sans-serif;
}
QPushButton:hover {
    background-color: #4F46E5;
    /* Hover efektinde hafif gölge (isteğe bağlı) */
}

QPushButton:pressed {
    background-color: #4338CA;
    padding-top: 14px; /* Tıklama hissi verir */
}</string>
                 </property>
                 <property name="text">
                  <string>Kelime Kutusu</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_exercise">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #6366F1; /* Modern İndigo Mavisi */
    color: white;
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
    font-size: 14px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, sans-serif;
}
QPushButton:hover {
    background-color: #4F46E5;
    /* Hover efektinde hafif gölge (isteğe bağlı) */
}

QPushButton:pressed {
    background-color: #4338CA;
    padding-top: 14px; /* Tıklama hissi verir */
}</string>
                 </property>
                 <property name="text">
                  <string>Alıştırmalar</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_update_pass">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: #6366F1; /* Mevcut mor/mavi tonu */
    color: white;
    border-radius: 12px;
    padding: 10px;
    font-weight: bold;
}</string>
                 </property>
                 <property name="text">
                  <string>Profil güncelle</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_delete_acc">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#btn_delete_acc {
    background-color: #e74c3c; /* Silme butonu için dikkat çekici kırmızı */
}</string>
                 </property>
                 <property name="text">
                  <string>Hesap sil</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>148</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QStackedWidget" name="stackedWidget_Content">
              <property name="styleSheet">
               <string notr="true">QListWidget {
    background-color: white;
    border: 1px solid #E2E8F0;
    border-radius: 12px;
    outline: none;
    padding: 5px;
}

QListWidget::item {
    padding: 10px;
    border-bottom: 1px solid #F1F5F9;
}

QListWidget::item:selected {
    background-color: #EEF2FF;
    color: #6366F1;
    border-radius: 8px;
}</string>
              </property>
              <widget class="QWidget" name="page">
               <widget class="QFrame" name="frame">
                <property name="geometry">
                 <rect>
                  <x>-210</x>
                  <y>3</y>
                  <width>866</width>
                  <height>561</height>
                 </rect>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2"/>
               </widget>
              </widget>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuDosya"/>
  </widget>
  <action name="actionLogout">
   <property name="text">
    <string>Oturumu kapat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
